projects/Mallard/src/screens/article/dev-tools.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react'
+
+import { articlePillars, ArticleType } from 'src/common'
+import { StyleSheet, View } from 'react-native'
+import { metrics } from 'src/theme/spacing'
+import { Button, ButtonAppearance } from 'src/components/button/button'
+
+export const getEnumPosition = <T extends {}>(value: any, position: number) => {
```

Comment:
This type signature is not type safe and I'm not sure what use the function is. If it's required to enable use of string enums for `ArticleType` this should just be a string union instead.